src/constants/custom-icons.js

```diff
@@ -0,0 +1,12 @@
+const iconmoonFont = 'icomoon';
```

Comment:
Undefined Font. Did we check before submitting ?

Reply:
now its working